src/views/webviews/welcome/welcome.ts

```diff
@@ -0,0 +1,165 @@
+import { ExtensionContext, ViewColumn, Webview, window, Uri } from 'vscode'
+
+const WelcomeView = {
+  title: 'Welcome to Pullflow',
+  type: 'welcome-view',
+}
+
+export const Welcome = {
+  show: (context: ExtensionContext) => {
+    const column = window.activeTextEditor?.viewColumn || ViewColumn.One
+    const panel = window.createWebviewPanel(
+      WelcomeView.type,
+      WelcomeView.title,
+      column
+    )
+    panel.iconPath = Uri.joinPath(context.extensionUri, 'assets/pullflow.png')
+    panel.webview.html = Welcome.getHtml(panel.webview, context)
+    panel.reveal(column)
+  },
+
+  getHtml: (webview: Webview, context: ExtensionContext) => {
+    const assetsPath = webview.asWebviewUri(
+      Uri.joinPath(context.extensionUri, 'assets')
+    )
+    const stylePath = webview.asWebviewUri(
+      Uri.joinPath(context.extensionUri, 'styles/welcome.css')
+    )
+
+    return `<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+      <title>Welcome to Pullflow</title>
+
+      <link rel="stylesheet" href=${stylePath} />
+    </head>
+
+    <body>
+      <header>
+        <img src="${assetsPath}/pullflow-logo.png" alt="Pullflow" />
+        <div id="desc">
+          <span>Code review collaboration across GitHub, Slack, and VS Code.</span>
+        </div>
+      </header>
+      <section class="container">
+        <section class="column">
+
+          <div id="intro">
+            <p> Pullflow is an AI-enhanced code review collaboration platform backed by the co-founder of GitHub and used by
+              some of the most innovative dev teams, including <strong> Epic Games, Avenue, Hear.com,
+                and RedwoodJS. </strong> </p>
+            <p>
+              It offers seamless integration with your <strong>Slack</strong> and <strong>GitHub </strong> accounts. Simply
+              authenticate your accounts within Pullflow, and the
+              synchronization will be automatically established. New to Pullflow? <a href="https://app.pullflow.com"> Sign
+                In </a>
```

Comment:
```suggestion
              ```suggestion
  Sign in to your accounts with Pullflow to enable two-sync between GitHub and Slack. New to Pullflow? <a href="https://pullflow.com">Sign up for free.</a>
```